core/.js/src/main/scala/io/udash/bindings/inputs/Input.scala

```diff
@@ -56,3 +56,13 @@ object PasswordInput extends Input("password")
 
 /** Number input. (HTML5) */
 object NumberInput extends Input("number")
+
+/** Datetime input (Doesn't work on firefox - falls back to text input). (HTML5) */
+object DateTimepicker extends Input("datetime-local")
+
+/** Date input. (HTML5) */
+object Datepicker extends Input("date")
+
+/** Time input. (HTML5) */
+object Timepicker extends Input("time")
+
```

Comment:
Any reason to rename those to "pickers" in such a low level API? They're still inputs according to docs.

Reply:
I don't understand I'm afraid